src/bootstrap/README.md

```diff
@@ -111,6 +111,42 @@ compiler. What actually happens when you invoke rustbuild is:
 The goal of each stage is to (a) leverage Cargo as much as possible and failing
 that (b) leverage Rust as much as possible!
 
+## Incremental builds
+
+You can configure rustbuild to use incremental compilation. Because
+incremental is new and evolving rapidly, if you want to use it, it is
+recommended that you replace the snapshot with a locally installed
+nightly build of rustc. You will want to keep this up to date.
+
+To follow this course of action, first thing you will want to do is to
+install a nightly, presumably using `rustup`. You will then want to
+configure your directory to use this build, like so:
+
+```
+# configure to use local rust instead of downloding a beta.
+# `--local-rust-root` is optional here. If elided, we will
+# use whatever rustc we find on your PATH.
+> configure --enable-rustbuild --local-rust-root=~/.cargo/ --enable-local-rebuild
```

Comment:
Aha! I found out [where we actually read this](https://github.com/rust-lang/rust/blob/908dba0c9477b7dd022a236cb1514ddfca9369f2/src/bootstrap/bootstrap.py#L229). Ideally we wouldn't need `--local-rust-root` here because `--enable-local-rebuild` should say "get from PATH by default".

Let's fix that bug later.